src/plugins/data/public/plugin.ts

```diff
@@ -162,7 +166,7 @@ export class DataPublicPlugin implements Plugin<DataPublicPluginSetup, DataPubli
 
     const dataServices = {
       actions: {
-        createFiltersFromEvent,
+        createFiltersFromValueClickAction,
```

Comment:
Can we please also export `createFiltersFromRangeSelectAction` in this pr? will be used by dashboard to dashboard Drilldowns

Reply:
@ppisljar, I guess we need to add it in more places to make it accessible.
here: src/plugins/data/public/types.ts
and then data/mock.ts / karma mocks
and this could also will fail on type check after src/legacy/core_plugins/vis_type_vislib/public/plugin.ts